pyClient/zeth/joinsplit.py

```diff
@@ -53,6 +86,26 @@ def __init__(self, a_pk: OwnershipPublicKey, k_pk: EncryptionPublicKey):
         self.a_pk: OwnershipPublicKey = a_pk
         self.k_pk: EncryptionPublicKey = k_pk
 
+    def __str__(self) -> str:
+        """
+        Write the address as "<ownership-key-hex>:<encryption_key_hex>".
+        (Technically the ":" separator is not required, since the first key is
+        written with fixed length, but it is provides a small amount amount of
+        sanity checking).
```

Comment:
but it is provides a small amount amount of sanity checking -> can you rephrase this?

Reply:
rephrased this